telegram_scraper.py

```diff
@@ -103,6 +108,20 @@ def main(
         help="The tracer id",
     )
 
+    parser.add_argument(
+        "--start_date",
```

Comment:
This is just a detail, but we were following the convention of putting `-` instead of `_` for command line arguments (i.e., those you'd write in your terminal when calling the scraper). I.e., to be consistent with the other CLI arguments, these should be `--start-date` and `--end-date`

Reply:
Note that you only need to change this here and make no other changes in the code. `argparse` will  automatically convert `--start-date` and so on to `args.start_date` in line 210